src/runtime/audience-action-flow.ts

```diff
@@ -321,6 +327,47 @@ export class AudienceActionFlow {
   }
 
   /**
+   * Populates localStorage with PPS configuration parameters based on
+   * SurveyDataTransferRequest.
+   **/
+  private async configureAnswerPpsData(
+    request: SurveyDataTransferRequest
+  ): Promise<void> {
+    const iabAudienceKey = StorageKeys.PPS_TAXONOMIES;
+    // PPS value field is optional and category may not be populated
+    // in accordance to IAB taxonomies.
+    const ppsConfigParams = request
+      .getSurveyQuestionsList()!
+      .flatMap((question) => question.getSurveyAnswersList())
+      .map((answer) => answer?.getPpsValue())
+      .filter((ppsValue) => ppsValue !== null);
+
+    const existingIabTaxonomy = await this.storage_.get(
+      iabAudienceKey,
+      /* useLocalStorage= */ true
+    );
+    const existingIabTaxonomyValues = existingIabTaxonomy
+      ? JSON.parse(existingIabTaxonomy)[Constants.PPS_AUDIENCE_TAXONOMY_KEY]
+          .values
+      : [];
+    const iabTaxonomyValues = Array.from(
+      new Set(ppsConfigParams.concat(existingIabTaxonomyValues))
+    );
+    const existingIabTaxonomyMap = {
```

Comment:
nit: it's not existing, but something new to be store. What about `iabTaxonomy` or `ppsTaxonomy`?